### ISSUE TITLE
tf-slim ResNet v1 pre-trained models preprocessing

### ISSUE THREAD

**Issue by neuralmagic** [tensorflow/tensorflow#41643]
### Affected Doc URL
https://github.com/tensorflow/models/blob/master/research/slim/README.md#pre-trained-models

### Description

Both tf-slim VGG and Inception preprocessing cause the ResNet v1 models in the above link to have incorrect outputs.  The other ImageNet models run correctly with either the VGG or Inception preprocessing, however under the same code path, ResNet v1 models produce incorrect outputs.  This issue is also documented in [this unresolved issue](https://github.com/tensorflow/tensorflow/issues/17426).

Are we aware of the correct steps to take to correctly run slim's ResNet v1 models and could this be updated in the documentation?

Thank you.

**Comment by neuralmagic** [comment]
Hi @ymodak we have found that the model expects inputs not to be scaled by 255, however this does not get the expected top1 accuracy (67% vs 75% on resnet v1 50).  Could you point us to any information or documentation on the proper preprocessing?  Thank you.

**Comment by ymodak** [comment]
@neuralmagic I think this issue is better raised on [tf-slim repo](https://github.com/google-research/tf-slim/issues) since the  [tf-slim VGG and Inception preprocessing](https://github.com/tensorflow/models/tree/master/research/slim/preprocessing) code is hosted on `tf-slim`.
The folks of `tf-slim` can provide a better guidance in this regard.

**Comment by neuralmagic** [comment]
Thank you @ymodak.  We will follow up there.
